change/@fluentui-react-virtualizer-701a2cd4-bd8c-48af-a2d4-753903d62124.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33125&commitId=c399aab79ae1132080766e0bca213644d993fd53&env=prod&clientType=FLUENTUI)


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2219| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2829| Changed|
|Drawer.overlay drawer full.chromium.png|4795| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>